### PR TITLE
KEYCLOAK-10440 Update Japanese translation for version 7.0.1

### PR DESCRIPTION
**Pull request by k-tamura**
Hi Keycloak developer team,

Could you assign @wadahiro as a reviewer?

### CODE REVIEW

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 1]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+authenticatorFinishSetUpMessage=Keycloakアカウントにサインインするたびに、2要素認証コードを入力するように求められます。
+authenticatorSubTitle=2要素認証を設定する
+authenticatorSubMessage=アカウントのセキュリティーを強化するには、利用可能な2要素認証の方式のうち少なくとも1つを有効にします。
+authenticatorMobileTitle=モバイル・オーセンティケーター
+authenticatorMobileMessage=モバイル・オーセンティケーターを使用して、2要素認証として確認コードを取得します。
+authenticatorMobileFinishSetUpMessage=オーセンティケーターはあなたの携帯電話にバインドされています。
+authenticatorActionSetup=セットアップ
+authenticatorSMSTitle=SMSコード
+authenticatorSMSMessage=Keycloakは、2要素認証として確認コードを携帯電話に送信します。
+authenticatorSMSFinishSetUpMessage=送信されるテキスト・メッセージ
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura I think `Text messages are sent to` tries to tell that the message(verification code) are sent to your mobile phone. How about `テキスト・メッセージが次の電話番号宛に送信されます：` ?
However, I think this message isn't used in account app (preview version) currently.

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 2]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+mobileSetupStep2=アプリケーションを開き、バーコードをスキャンしてください。
+mobileSetupStep3=アプリケーションから提供されたワンタイムコードを入力し、保存をクリックしてセットアップを終了します。
+scanBarCode=バーコードをスキャンしますか？
+enterBarCode=ワンタイムコードを入力してください
+doCopy=コピー
+doFinish=終了
+
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=SMSコードのセットアップ
+smscodeIntroMessage=電話番号を入力すると、確認コードがあなたの電話に送信されます。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L308

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 3]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+authenticatorMobileMessage=モバイル・オーセンティケーターを使用して、2要素認証として確認コードを取得します。
+authenticatorMobileFinishSetUpMessage=オーセンティケーターはあなたの携帯電話にバインドされています。
+authenticatorActionSetup=セットアップ
+authenticatorSMSTitle=SMSコード
+authenticatorSMSMessage=Keycloakは、2要素認証として確認コードを携帯電話に送信します。
+authenticatorSMSFinishSetUpMessage=送信されるテキスト・メッセージ
+authenticatorDefaultStatus=デフォルト
+authenticatorChangePhone=電話番号の変更
+authenticatorBackupCodesTitle=バックアップ・コード
+authenticatorBackupCodesMessage=8桁のバックアップコードの入手
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 4]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+authenticatorBackupCodesFinishSetUpMessage=この時点で12個のバックアップコードが生成されました。それぞれ一度だけ使用できます。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 5]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+#Authenticator - SMS Code setup
+authenticatorSMSCodeSetupTitle=SMSコードのセットアップ
+smscodeIntroMessage=電話番号を入力すると、確認コードがあなたの電話に送信されます。
+chooseYourCountry=国を選んでください
+enterYourPhoneNumber=電話番号を入力してください
+sendVerficationCode=確認コードの送信
+enterYourVerficationCode=確認コードを入力してください
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=バックアップコードのセットアップ
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 6]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+backupcodesIntroMessage=携帯電話にアクセスできない場合でも、バックアップコードを使用してアカウントにログインできます。どこか安全でアクセス可能な場所に保管してください。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 7]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+realmName=レルム
+doDownload=ダウンロード
+doPrint=印刷
+doCopy=コピー
+backupCodesTips-1=各バックアップコードは1回使用できます。
+backupCodesTips-2=これらのコードはこの日に生成されました：
+generateNewBackupCodes=新しいバックアップコードを生成する
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 8]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+enterYourVerficationCode=確認コードを入力してください
+
+#Authenticator - backup Code setup
+authenticatorBackupCodesSetupTitle=バックアップコードのセットアップ
+backupcodesIntroMessage=携帯電話にアクセスできない場合でも、バックアップコードを使用してアカウントにログインできます。どこか安全でアクセス可能な場所に保管してください。
+realmName=レルム
+doDownload=ダウンロード
+doPrint=印刷
+doCopy=コピー
+backupCodesTips-1=各バックアップコードは1回使用できます。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 9]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+backupCodesTips-2=これらのコードはこの日に生成されました：
+generateNewBackupCodes=新しいバックアップコードを生成する
+backupCodesTips-3=新しいバックアップコードを生成すると、現在のコードは機能しなくなります。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura バックアップ・コード

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 10]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+backtoAuthenticatorPage=認証ページに戻る
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura オーセンティケーター・ページに戻る

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 11]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+
+
+#Resources
+resources=リソース
+myResources=私のリソース
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L330

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 12]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+sharedwithMe=私と共有
+share=共有
+resource=リソース
+application=アプリケーション
+date=日付
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L333-L335

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 13]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+sharedwith=共有
+owner=オーナー
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L337

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties [hunk 14]
```diff
@@ -255,3 +263,99 @@ allApps=すべてのアプリケーション
...
+accessPermissions=アクセス・パーミッション
+permissionRequests=パーミッション・リクエスト
+approve=承認
+approveAll=すべて承認
+sharedwith=共有
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: @k-tamura This is duplicate message. The message was removed from English resource.
https://github.com/keycloak/keycloak/commit/f35d3a8ec9a0c404d6e92c19babc520843ead368#diff-212b66440fb033927a3a24caf41983c9L342

**Review by wadahiro — CHANGES_REQUESTED** [review]
@k-tamura I checked `themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties`. Please check my comments.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 15]
```diff
@@ -107,8 +112,21 @@ days=日
...
+sso-session-max-remember-me=SSOセッション最大リメンバー・ミー
+sso-session-max-remember-me.tooltip=ユーザーがリメンバー・ミー・オプションを設定したときにセッションが期限切れになるまでの最大時間。セッションが期限切れになると、トークンおよびブラウザー・セッションは無効になります。設定されていない場合は、標準のSSO Session Max値が使用されます。
 offline-session-idle=オフラインセッションアイドル
 offline-session-idle.tooltip=セッションの有効期限が切れるまでのオフライン時間です。この期限内に少なくとも1回はオフライントークンを使用してリフレッシュしないと、オフラインセッションは有効期限切れとなります。
+realm-detail.hostname=ホスト名
+realm-detail.hostname.tooltip=レルムに対してホスト名を設定します。特定のレルムのサーバーホスト名を上書きするために、固定ホスト名プロバイダーと組み合わせて使用​​します。
+
+## KEYCLOAK-7688 Offline Session Max for Offline Token
+offline-session-max-limited=オフラインセッション最大制限
+offline-session-max-limited.tooltip=オフラインセッションを有効にします。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: @k-tamura オフラインセッションの最大時間制限を有効にします。

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 16]
```diff
@@ -193,22 +213,25 @@ userSession.modelNote.label=ユーザーセッションノート
 userSession.modelNote.tooltip=UserSessionModel.note マップ内のユーザーセッションノート名です。
 multivalued.label=マルチバリュー
 multivalued.tooltip=属性がマルチバリューをサポートしているかどうかを示します。サポートしている場合は、この属性のすべての値リストがクレームとして設定されます。サポートしていない場合は、最初の値だけがクレームとして設定されます。
+aggregate.attrs.label=属性値の集約
+aggregate.attrs.tooltip=属性値をグループ属性と集約する必要があるかどうかを示します。 OpenID Connectマッパーを使用している場合は、すべての値を取得するためにマルチバリューのオプションも有効にする必要があります。重複した値は破棄され、値の順序はこのオプションでは保証されません。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: @k-tamura Remove space from `します。 OpenID`

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 17]
```diff
@@ -310,6 +337,14 @@ web-origins=Web オリジン
...
 fine-oidc-endpoint-conf=OpenID Connect の詳細設定
 fine-oidc-endpoint-conf.tooltip=このセクションを展開して、OpenID Connect プロトコルに関連するこのクライアントの高度な設定を行います。
+access-token-signed-response-alg=アクセストークン署名アルゴリズム
+access-token-signed-response-alg.tooltip=アクセストークンの署名に使用されるJWAアルゴリズム。
+id-token-signed-response-alg=IDトークン署名アルゴリズム
+id-token-signed-response-alg.tooltip=IDトークンの署名に使用されるJWAアルゴリズム。
+id-token-encrypted-response-alg=IDトークン暗号化鍵管理アルゴリズム
+id-token-encrypted-response-alg.tooltip=IDトークンの暗号化鍵の管理に使用されるJWAアルゴリズム。このオプションは、暗号化されたIDトークンが必要な場合にのみ必須です。空のままにすると、IDトークンは署名されますが、暗号化されません。
+id-token-encrypted-response-enc=IDトークン暗号化コンテンツの暗号化アルゴリズム
+id-token-encrypted-response-enc.tooltip=id-token-encrypted-response-enc.tooltip=IDトークンの暗号化の際に、コンテンツの暗号化に使用されるJWAアルゴリズム。このオプションは、暗号化されたIDトークンが必要な場合にのみ必須です。空のままにすると、IDトークンは署名されますが、暗号化されません。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: @k-tamura Remove `id-token-encrypted-response-enc.tooltip=` ? It looks like original English message has same issue.
https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/admin/messages/admin-messages_en.properties#L349

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 18]
```diff
@@ -1491,4 +1553,8 @@ manage-members-authz-group-scope-description=管理者がこのグループの
 advanced-client-settings=詳細設定
 advanced-client-settings.tooltip=このセクションを展開して、このクライアントの詳細設定を設定します
 tls-client-certificate-bound-access-tokens=OAuth 2.0相互TLS証明書バインドアクセストークンが有効
-tls-client-certificate-bound-access-tokens.tooltip=これにより、OAuth 2.0相互TLS証明書バインドアクセストークンがサポートされます。つまり、Keycloakは、Keycloakのトークンエンドポイントとこのクライアントの間で相互TLSにより交換されるクライアントのX.509証明書と、アクセストークンおよびリフレッシュトークンをバインドします。これらのトークンは、ベアラトークンの代わりにHolder-of-Keyトークンとして扱うことができます。
\ No newline at end of file
+tls-client-certificate-bound-access-tokens.tooltip=これにより、OAuth 2.0相互TLS証明書バインドアクセストークンがサポートされます。つまり、Keycloakは、Keycloakのトークンエンドポイントとこのクライアントの間で相互TLSにより交換されるクライアントのX.509証明書と、アクセストークンおよびリフレッシュトークンをバインドします。これらのトークンは、ベアラトークンの代わりにHolder-of-Keyトークンとして扱うことができます。
+subjectdn=サブジェクトDN
+subjectdn-tooltip=クライアント証明書内のサブジェクトDNを検証するための正規表現。あらゆる種類の式に一致させるには、"(.*?)(?:$)"を使用します。
+pkce-code-challenge-method=コード交換用証明鍵のコードチャレンジ方式
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: @k-tamura `Proof Key for Code Exchangeのコードチャレンジ方式`

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 19]
```diff
@@ -1491,4 +1553,8 @@ manage-members-authz-group-scope-description=管理者がこのグループの
...
+pkce-code-challenge-method.tooltip=PKCEにどのコードチャレンジ方式を使用するかを選択します。指定しない場合、keycloakは、クライアントが適切なコードチャレンジとコード交換の方式で認可リクエストを送信しない限り、クライアントにPKCEを適用しません。
```
Comment (wadahiro) [inline themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: @k-tamura `PKCEに` -> `PKCEの`

**Review by wadahiro — CHANGES_REQUESTED** [review]
@k-tamura I finished checking remains message resources. Please check my comments.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties (hunk 17 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: Thanks. I will fix it too: https://github.com/keycloak/keycloak/pull/6480

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties (hunk 2 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: Oh! I completely forgot...

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties (hunk 3 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: Okay.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties [hunk 20]
```diff
@@ -107,8 +112,21 @@ days=日
 sso-session-max=SSO セッション最大
 sso-session-idle.tooltip=セッションの有効期限が切れるまでのアイドル時間です。セッションが有効期限切れの際はトークンとブラウザーセッションは無効化されます。
 sso-session-max.tooltip=セッションの有効期限が切れるまでの最大時間です。セッションが有効期限切れの際はトークンとブラウザーセッションは無効化されます。
+sso-session-idle-remember-me=SSOセッション・アイドル・リメンバー・ミー
+sso-session-idle-remember-me.tooltip=セッションが期限切れになるまでのアイドル状態であることを許可されている時間。セッションが期限切れになると、トークンおよびブラウザー・セッションは無効になります。設定されていない場合は、標準のSSOセッション・アイドル値が使用されます。
```
[the remark this answers is not in the document]
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: I agree with you.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties (hunk 15 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: I see.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties (hunk 16 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: Okay.

File: themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties (hunk 18 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/admin/messages/admin-messages_ja.properties]: I overdid it...

**Comment by stianst** [comment]
@wadahiro  can you review again? @k-tamura has updated the PR.

**Comment by stianst** [comment]
@k-tamura @wadahiro any update on this one?

File: themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties (hunk 5 above)
Comment (k-tamura) [reply themes/src/main/resources-community/theme/base/account/messages/messages_ja.properties]: I will change all words like these as other issue.

**Review by wadahiro — DISMISSED** [review]
@stianst LGTM. Could you merge this?

**Review by stianst — APPROVED** [review]

**Pull request merged**